lib/commands/users.js

```diff
@@ -2009,11 +2236,17 @@ module.exports = {
     usersListLogs,
     usersListMemberships,
     usersUpdateMfa,
+    usersUpdateMFA,
     usersDeleteMfaAuthenticator,
+    usersDeleteMFAAuthenticator,
```

Comment:
Notice double definitions for the same here, I doublt that will be valid JS code. We might want manual QA here as well, to prevent problems like those. We dont have any CI/CD tests for CLI